src/components/form-helpers/form-helpers.js

```diff
@@ -308,13 +312,11 @@ const _PhoneFieldItem = ({
     });
     inputElRef.current.addEventListener('countrychange', handleChange);
     intlTelInputRef.current = iti;
-    // It is to force international number on reload by language change, in another case
-    // it uses national mode because of nationalMode
-    formatFieldValueAsInternationalNumber();
+    _formatFieldValueAsInternationalNumber(iti, fieldName, setFieldValue);
     return () => {
       iti.destroy();
     };
-  }, [intl.locale]);
+  }, [intl.locale, handleChange, fieldName, setFieldValue]);
```

Comment:
The new validation done in the Travis build process detected this issue. Probably, it was working fine most of the times, but I like having this lint rule.

As I have [mentioned in another comment](https://github.com/FromDoppler/doppler-webapp/pull/250#discussion_r294931584), we should validate this kind of issues in our Jenkins build process.